### PR TITLE
Update image to build for Python 3.8

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+	  gcc \
+	  python38 \
+	  python38-devel \
+	  jq \
+	  nano \
+	  unzip \
+	  zip \
+	  && yum clean all
+
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 1
```
Comment (akarve) [inline Dockerfile]: Maybe we want a default higher priority.
```suggestion
RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 10
```

**Review by sir-sigurd — COMMENTED** [review]
Can't we use one of [these](https://docs.aws.amazon.com/lambda/latest/dg/python-image.html#python-image-base) or [these](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-image-repositories.html#serverless-image-repository-uris) images?

**Comment by akarve** [comment]
> Can't we use one of [these](https://docs.aws.amazon.com/lambda/latest/dg/python-image.html#python-image-base) or [these](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-image-repositories.html#serverless-image-repository-uris) images?

Done and that's a better solution. I actually looked for these images and didn't find them at first. Googling "amazon lambda docker image" is not very useful but "amazon lambda docker *hub* image" gets you to the right place. Happens to be a separate repo from amazon-linux 2.

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-s3transfer==0.3.3
-six==1.14.0
-urllib3==1.25.8
+# from `pip freeze` under Python 3.8 lambda image 01-Sep-2021
+boto3==1.17.100
+botocore==1.20.100
+Cython==0.29.24
+jmespath==0.10.0
+python-dateutil==2.8.2
+rapid-client==0.0.0
```
Comment (dimaryaz) [inline requirements.txt]: This looks sketchy? No information at all about that package.

**Review by dimaryaz — COMMENTED** [review]
Looks fine, assuming it works - but curious where rapid-client came from.

File: requirements.txt (hunk 2 above)
Comment (akarve) [reply requirements.txt]: That's what I get for trusting Amazon. They install it. But on PyPI this is one of the sketchiest packages I've ever seen https://pypi.org/project/rapid-client/#files. The Homepage is example.com! WTH?
```
b'boto3==1.17.100\nbotocore==1.20.100\njmespath==0.10.0\npython-dateutil==2.8.2\nrapid-client==0.0.0\ns3transfer==0.4.2\nsix==1.16.0\nurllib3==1.26.6\n'
```
I'm gonna report this to AWS.

**Review by dimaryaz — APPROVED** [review]

**Pull request merged**